app/javascript/packages/document-capture/components/location-collection-item.spec.tsx

```diff
@@ -59,4 +59,28 @@ describe('LocationCollectionItem', () => {
       .parentElement!;
     expect(sunHours.textContent).to.contain('Closed');
   });
+
+  it('renders the component that includes contact information with expected data', () => {
+    const onClick = sinon.stub();
+    const { getByText } = render(
+      <LocationCollectionItem
+        distance="1.0 mi"
+        phone="555-123-4567"
+        tty="222-222-2222"
+        name=""
+        streetAddress="123 Test Address"
+        formattedCityStateZip=""
+        handleSelect={onClick}
+        weekdayHours=""
+        saturdayHours=""
+        selectId={0}
+        sundayHours=""
+      />,
+    );
+
+    const addressParent = getByText('123 Test Address').parentElement!;
+    expect(addressParent.textContent).to.contain('in_person_proofing.body.location.distance');
+    expect(addressParent.textContent).to.contain('555-123-4567');
+    expect(addressParent.textContent).to.contain('222-222-2222');
```

Comment:
Any reason not to merge this test with the one above? They're very similar already

Reply:
not really, i'll adjust it so its one test

Reply:
oh wait, they do have to be separate as locationCollectionItem can display either name or distance but not both. when the po search page goes into production though we should remove the above test as we'd just display the distance and not the po name.